src/NuGet.Core/NuGet.Commands/PackCollectorLogger.cs

```diff
@@ -67,11 +75,32 @@ public override Task LogAsync(ILogMessage message)
             return Task.FromResult(0);
         }
 
+        /// <summary>
+        /// This method checks if at least one of the warning properties collections is not null and it suppresses the warning.
+        /// </summary>
+        /// <param name="message">IRestoreLogMessage to be logged.</param>
```

Comment:
```suggestion
        /// <param name="message">ILogMessage to be logged.</param>
```

Alternatively, this is a private method, you don't need the docs.

Reply:
Removed docs for private methods.